andela/activo-web
# #158354354 Launch add asset form

#### What does this PR do?
Creates the functionality for launching the add asset form.

#### Description of tasks to be completed?
- write method handler for changing the class of the continue button
- write method handler for listening for change events on the select tag
- write method handler for displaying asset form
- modify addassetpane.scss file to reflect new changes

#### How should this be manually tested?
- Run yarn start:dev to start the dev server.
- Sign in with your Andela account. Click on the caret close to the displayed username.
- Click on the `Add Asset` button to open the Add asset pane.
- The `Continue` button should be disabled.
- Select an option, and the continue button should now be active.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#158354354](https://www.pivotaltracker.com/story/show/158354354)

#### Screenshots
<img width="1280" alt="screen shot 2018-06-26 at 9 21 16 am" src="https://user-images.githubusercontent.com/24798364/41898599-58726d56-7922-11e8-8eae-733b4ebae2e8.png">

<img width="1280" alt="screen shot 2018-06-26 at 9 21 22 am" src="https://user-images.githubusercontent.com/24798364/41898736-a7de78bc-7922-11e8-87ea-9214ac331643.png">
